### PR TITLE
Logout from all devices link

### PR DESCRIPTION
**Pull request by lookacat**
## Description
We've added "Logout from active devices" in the account settings

## Related Issue

- Fixes https://github.com/owncloud/web/issues/8609

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/web-pkg/src/configuration/manager.ts [hunk 1]
```diff
@@ -34,6 +35,15 @@ export class ConfigurationManager {
     this.oidcConfiguration = rawConfig.openIdConnect
       ? (rawConfig.openIdConnect as OIDCConfiguration)
       : null
+    this.logoutUrl = rawConfig.logoutUrl ? rawConfig.logoutUrl : null
```
[the remark this answers is not in the document]
Comment (lookacat) [reply packages/web-pkg/src/configuration/manager.ts:38]: no, you are right

File: packages/web-runtime/tests/unit/pages/account.spec.ts [hunk 2]
```diff
@@ -23,6 +23,12 @@ const selectors = {
   groupNamesEmpty: '[data-testid="group-names-empty"]'
 }
 
+jest.mock('web-pkg/src/configuration', () => ({
+  configurationManager: {
+    logoutUrl: "https://owncloud.com"
```
[the remark this answers is not in the document]
Comment (lookacat) [reply packages/web-runtime/tests/unit/pages/account.spec.ts]: yes

File: packages/web-pkg/src/configuration/manager.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ export interface RawConfig {
   auth?: any
   openIdConnect?: any
   options?: OptionsConfiguration
+  logoutUrl?: string
```
Comment (kulmann) [inline packages/web-pkg/src/configuration/manager.ts:17]: Please make the `logoutUrl` part of the `OptionsConfiguration` for now. Adding new config in oCIS is a pain, so we just go with putting everything in `options` for the moment.

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/web-pkg/src/configuration/manager.ts (hunk 3 above)
Comment (lookacat) [reply packages/web-pkg/src/configuration/manager.ts:17]: alright 👍🏼

File: packages/web-runtime/src/pages/account.vue [hunk 4]
```diff
@@ -78,6 +80,21 @@
...
+        <dd data-testid="logout">
+          <oc-button
+            appearance="raw"
+            type="a"
+            :href="logoutUrl"
+            target="_blank"
+            data-testid="account-page-logout-url-btn"
+          >
+            <span v-text="$gettext('Account security')" />
+            <oc-icon name="edit" />
```
Comment (kulmann) [inline packages/web-runtime/src/pages/account.vue:94]: what's the `edit` icon for? 🤔

Comment (lookacat) [reply packages/web-runtime/src/pages/account.vue:94]: oh right, thats not correct

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/34067/25/1
:boom: The oC10Sharing2 tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/34067/23/1
:boom: The oC10SharingAccept tests pipeline failed. The build has been cancelled.

**Pull request merged**